src/styles/blocks/catalog.scss

```
@@ -17,6 +18,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $smoothly-transform;
```

It's better to specify properties you want to add an effect to
```suggestion
    transition: color $smoothly-transform, transform $smoothly-transform;
```